app/models/user.rb

```diff
@@ -36,9 +36,11 @@ def core_induction_programme
     mentor_profile.core_induction_programme if mentor?
   end
 
-  def is_on_full_induction_programme?
-    early_career_teacher_profile&.full_induction_programme? ||
-      mentor_profile&.full_induction_programme?
+  def is_not_on_core_induction_programme?
+    is_on_full_induction_programme? ||
+      is_on_designed_programme? ||
+      programme_has_no_early_career_teachers? ||
+      programme_not_yet_known?
   end
```

Comment:
`! is_on_core_induction_programme?`?

Reply:
I tried this but didn't have luck, i'll have another look at it

Reply:
So it was causing non ect or mentor profiles to also be denied 

Doing  `!is_on_core_induction_programme? && (mentor? || early_career_teacher?)` looks to work fine